package.json

```diff
@@ -13,17 +13,26 @@
     "type": "git",
     "url": "https://github.com/meisterplayer/media-nativehls.git"
   },
+  "scripts": {
+    "lint": "eslint ./src/js",
+    "test": "jest",
+    "test:coverage": "jest --coverage",
+    "build": "gulp build",
+    "dist": "gulp build:min && gulp build:dist"
+  },
   "author": "Triple",
   "license": "Apache-2.0",
   "dependencies": {},
   "devDependencies": {
-    "meister-gulp-webpack-tasks": "^1.0.6",
-    "meister-js-dev": "^3.1.0",
+    "@meisterplayer/meister-mock": "^1.0.0",
     "babel-preset-es2015": "^6.24.0",
     "babel-preset-es2017": "^6.22.0",
-    "gulp": "^3.9.1"
+    "gulp": "^3.9.1",
+    "jest": "^20.0.4",
+    "meister-gulp-webpack-tasks": "^1.0.6",
```

Comment:
It's not recommended to use a carrot with gulp-webpacktasks since this one broke semver compatibility...